launch/teleop.launch

```diff
@@ -1,15 +1,16 @@
 <launch>
 	<arg name="speed_linear" default="0.2" />
 	<arg name="speed_angular" default="1.0" />
-	<arg name="cmd_vel_topic" default="/miro/control/cmd_vel" />
+	<arg name="cmd_vel_topic" default="/miro/rob01/control/cmd_vel" />
```

Comment:
Because this breaks this for Miro-E (commands aren't stamped), can you make it a new file too.